src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -0,0 +1,495 @@
+        this.annotationNamespace = annotationNamespace;
+        this.validator = validator;
+        this.forceDump = forceDump;
+    }
+
+    /**
+     * Call before each test.  Returns a new, empty model with standard AIF namespaces.
+     *
+     * @return a new model with which to start a test; caller may wish to add prefixes for the ontology and annotation
+     */
+    Model startNewTest() {
+        if (model != null) {
+            typeAssertionJustification = null;
+            model.close();
+        }
+        model = ModelFactory.createDefaultModel();
+        // adding namespace prefixes makes the Turtle output more readable
+        addStandardNamespaces(model);
+        // every AIF needs an object for the system responsible for creating it
+        system = makeSystemWithURI(model, getTestSystemUri());
+        assertionCount = entityCount = eventCount = relationCount = hypothesisCount = clusterCount = documentCount = 1;
+
+        return model;
+    }
+
+    /**
+     * Returns a prefixed String system URI in case tests need to create their own models.
+     */
+    String getTestSystemUri() {
+        return getUri("testSystem");
+    }
+
+    /**
+     * Returns a prefixed String URI of the specified name.
+     */
+    String getUri(String localName) {
+        return annotationNamespace + localName;
+    }
+
+    /**
+     * Returns a unique prefixed String URI for use with assertions.
+     */
+    String getAssertionUri() {
+        return getUri("assertion-" + assertionCount++);
+    }
+
+    /**
+     * Returns a unique prefixed String URI for use with entities.
+     */
+    String getEntityUri() {
+        return getUri("entity-" + entityCount++);
+    }
+
+    /**
+     * Returns a unique prefixed String URI for use with events.
+     */
+    String getEventUri() {
+        return getUri("event-" + eventCount++);
+    }
+
+    /**
+     * Returns a unique prefixed String URI for use with relations.
+     */
+    String getRelationUri() {
+        return getUri("relation-" + relationCount++);
+    }
+
+    /**
+     * Returns a unique prefixed String URI for use with hypotheses.
+     */
+    String getHypothesisUri() {
+        return getUri("hypothesis-" + hypothesisCount++);
+    }
+
+    /**
+     * Returns a unique prefixed String URI for use with clusters.
+     */
+    String getClusterUri() {
+        return getUri("cluster-" + clusterCount++);
+    }
+
+    /**
+     * Returns the test system created by {@link #startNewTest()}.
+     */
+    Resource getSystem() {
+        return system;
+    }
+
+    /**
+     * Returns a unique document name.
+     */
+    String getDocumentName() {
+        return "document-" + documentCount++;
+    }
+
+    /**
+     * Returns a type assertion justification.
+     */
+    Resource getTypeAssertionJustification() {
+        if (typeAssertionJustification == null) {
+            typeAssertionJustification = makeTextJustification(model, getDocumentName(),
```

Comment:
Two reasons, one bad and one...maybe good?
The first reason is that `makeValidJustification` was added late in the game and I didn't update `getTypeAssertionJustification` to use it.  :)
The second reason is that I did kind of want the `typeAssertionJustification` to stand out a bit in the generated KB, so I kept the confidence at 0.973 and used offsets of 42 and 143.  In fact, to that end, maybe I should change the docId to something specific, like "typeAssertionDoc."  That said, it's kind of an artificial reason.  For that matter, we don't even really need `getTypeAssertionJustification` since the caller can now just call `makeValidJustification`.  I'm fine with any of these approaches.  Just let me know what you'd prefer.  Shall I just have it call `makeValidJustification` behind the scenes?

Reply:
Yeah, I'm not sure our previous methodology of having a bunch of static objects was the right approach. They were added because there wasn't a utility to create valid objects on the fly. Now that there IS this great utility, ideally, the NIST examples would make use of it and all the NIST outputs would be slimmed down to just the necessities. 

I know that changing the NIST examples is probably outside the scope of this ticket, but I did think that the creation of this utility would be a general way to make generating AIF easier. With that in mind, is there a benefit to all the type justifications being the same or is it just maintaining the status quo of having a single type justification?

Reply:
Changing the NIST examples to use the TestUtils API is in scope for this ticket, but changing them to produce better/optimal examples with less "extra stuff" is what [AIDA-720](https://nextcentury.atlassian.net/browse/AIDA-720) is about.

Like I said, having all of the type justifications makes it easier to recognize them in the generated examples, but I suspect many of you are better and reading straight AIF TTLs than I am.  ;)  I think what you're proposing is removing the `typeAssertionJustification` private member from the TestUtils class (including `getTypeAssertionJustifcation()`) and just have the examples call makeValidJustification whenever they need a justification, type assertion or otherwise.  Is this correct?  I'm on board with this, if that's what you're saying.

Reply:
Yeah, that's what I was thinking. Let me know if it presents any problems

Reply:
Done.